src/routes/FilterInterface.svelte

```diff
@@ -0,0 +1,143 @@
+<script>
+// @ts-nocheck
+
+  import { onMount } from 'svelte';
+  import { TreeView, TreeViewItem } from '@skeletonlabs/skeleton';
+
+  // Function to convert the tree data into a format suitable for rendering
+  /**
+	 * @param {{ [s: string]: any; } | ArrayLike<any>} data
+	 */
+  function convertDataToTreeView(data) {
+    return Object.entries(data).map(([group, values]) => {
+      if (Array.isArray(values) && values.length > 0) {
+        console.log('Leaf node:', group, values);
+        return { group, values };
+      } else if (values && typeof values === 'object' && Object.keys(values).length > 0) {
+        console.log('Parent node:', group);
+        const children = convertDataToTreeView(values);
+        return { group, children };
+      } else {
+        // Handle other cases
+        console.log('Node without values:', group);
+        return { group, children: [] };
+      }
+    });
+  }
+
+  // Fetch and process JSON data
+  let treeData;
+  let filePath = 'src/treeJson.json';
+  /**
+	 * @type {any[]}
+	 */
+  let treeViewData = [];
+
+  onMount(async () => {
+    try {
+      const response = await fetch(filePath);
+      treeData = await response.json();
+      console.log('Fetched treeData:', treeData);
+      treeViewData = convertDataToTreeView(treeData.mediums); // Assuming "mediums" is the root property
+      console.log('Converted treeViewData:', treeViewData);
+    } catch (error) {
+      console.error('Error loading JSON data:', error);
+    }
+  });
+
+  // Selected items that will be used later
+  /**
+	 * @type {never[]}
+	 */
+  let selectedItems = [];
+</script>
+<div class="dark-container w-screen flex pt-10 flex-col">
+  <h2 class ="h2 pl-16" style="text-align: left;" >Virtueller Katalog</h2>
+  <div class="side_by_side w-screen pl-2 pt-5 flex">
+    <div class="tree w-1/2 box-border text-sm">
+      <TreeView bind:selection={selectedItems} multiple>
```

Comment:
here the TS-Error actually shows a mistake you made!
This is a boolean prop. you cannot put in an array of something...

Reply:
```suggestion
      <TreeView selection multiple>
```